kornia/filters/bilateral.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+import numpy as np
+
+
+# gaussian filter
+def gfunc(x,y,sigma):
+    return (math.exp(-(x**2 + y**2)/(2*(sigma**2))))/(2*3.14*(sigma**2))
+
+def gaussFilter(size, sigma):
+    out = np.zeros(size)
+    for i in range(size[0]):
+        for j in range(size[1]):
+            out[i,j] = gfunc(i-size[0]//2,j-size[1]//2, sigma )
+    return out/np.sum(out)
+
+def bfunc(i,j,fw,fh,image,sigma1, bilateralWFilter):
```

Comment:
I believe, it would be better to implement the kernel (or couple of kernels) to be used with kornia filter2d
https://github.com/kornia/kornia/blob/master/kornia/filters/filter.py#L33

Reply:
agree to have separated function for this